.gitignore

```diff
@@ -7,6 +7,8 @@ tmp
 bin/configlet
 bin/configlet.exe
 .idea/
+.vscode/
+.mypy_cache/
```

Comment:
Please revert this change. Suggest adding to a [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) or your local `.git/info/exclude` file.

Reply:
But vscode is an IDE just like pycharm(.idea folder), and mypy cache comes from built-in mypy. Shoudn't it be included as some people code in vs code and might submit PR's from that too?

Reply:
I had no idea what `.idea/` was used for; that line in the `.gitignore` preceeds my involvement with the project.

General policy is to not include environment-specific content in the repository, but as this is a project that receives a lot of beginner contributors that may not know to not check in such files, I suppose we can make an exception for the gitignore file.